terraform/modules/ecs/variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "region" {
```

Comment:
Do we need this in ecs since we are using below one.This will set the current region from provider I think
data "aws_region" "current" {
}

Reply:
it didnt work when executed in workflow, and was throwing error as the "data" wasn't detected .